src/systemctl/systemctl-enable.c

```diff
@@ -263,6 +263,33 @@ int verb_enable(int argc, char *argv[], void *userdata) {
                            "  instance name specified.",
                            special_glyph(SPECIAL_GLYPH_BULLET));
 
+        if (streq(verb, "disable") && arg_scope == LOOKUP_SCOPE_USER) {
+                /* We emit the warning only if all the unit files are enabled in
```

Comment:
`all` ?? I think it is better when at least one.

Reply:
Hmm, I was thinking to use the same logic as the install info warning. But if otherwise is desirable, I'm happy to go with that :)